src/components/DatePicker/index.jsx

```diff
@@ -1,9 +1,232 @@
 import React, { Component } from 'react';
+import moment from 'moment';
+import InfiniteCalendar, {
+    Calendar,
+    withRange,
+} from 'react-infinite-calendar';
+import 'react-infinite-calendar/styles.css';
+import {EVENT_TYPE} from "react-infinite-calendar/es/Calendar/withRange";
+import Timekeeper from "react-timekeeper";
 
+const CalendarWithRange = withRange(Calendar);
+
+function getTheme() {
+    return {
+        accentColor: '#009384',
+        floatingNav: {
+            background: '#5B57A6',
+            chevron: '#FFF',
+            color: '#FFF',
+        },
+        headerColor: '#5B57A6',
+        selectionColor: '#009384',
+        textColor: {
+            active: '#FFF',
+            default: '#231F20',
+        },
+        todayColor: '#EEA616',
+        weekdayColor: '#5B57A6',
+    };
+}
+
+function getTimePickerTheme() {
```

Comment:
Directly have constants for these. No need to have a function to return an object without any logic. Better yet, have styles defined in a separate CSS file within this folder and use it.